src/seedu/addressbook/data/AddressBook.java

```diff
@@ -143,4 +167,19 @@ public UniquePersonList getAllPersons() {
     public UniqueTagList getAllTags() {
         return new UniqueTagList(allTags);
     }
+    
+    class Tagging {
+        
+        private ArrayList<String> trackTagsFromStart;
+        
```

Comment:
where are the Person and Tag objects between which you need to maintain the association?